docs/builtins.md

```diff
@@ -1256,3 +1256,463 @@ x-axis and the ray from the origin to the point `x y`. More formally,
 it returns the arc tangent of `y/x` for given arguments `y` and `x`,
 using the signs of the two to determine the quadrant of the return
 value.
+
+## Graphics
+
+`evy` on the web outputs drawing commands to a drawing area
+called **canvas**.
+
+Positions on the canvas are defined by a coordinate system, similar to
+the Cartesian coordinate system used in mathematics. The horizontal
+dimension is called the **x-axis**, and the vertical dimension is
+called the **y-axis**.
+
+A point on the canvas is defined by its **x** and **y coordinates**,
+which are written as `x y`. For example, the point `30 60` has an
+x-coordinate of 30 and a y-coordinate of 60. It is located 30 units
+from the left edge of the canvas and 60 units from the bottom edge.
+
+<img width="300" alt="Catesian coordinates with x and y axes and point (30, 60)" src="img/coords.png">
```

Comment:
s/Catesian/Cartesian/

Unless you really meant :cat:

Reply:
thanks!